public/locales/es/common.json

```diff
@@ -12,12 +12,16 @@
       "banner_3_btnLabel": "Saber más"
     },
     "testimonials": {
+      "title": "TESTIMONIOS",
+      "sub-title": "Nuestros clientes felices",
       "banner_1": "Cien por ciento mejor, salieron las manchas, limpieza profunda y un resultado perfecto",
       "banner_2": "1000% mejor, casi como nuevo",
       "banner_3": "Trabajo profesional, excelente trabajo de mármol muy brillante",
       "banner_4": "Limpieza de alfombra profesional, muy limpia y como nueva",
       "banner_5": "Stripper de piso profesional, brillante y muy limpio",
-      "banner_6": "Excelente Mantenimiento de piso un cepillado “scrubfloor” profundo todo muy limpio"
+      "banner_6": "Excelente Mantenimiento de piso un cepillado “scrubfloor” profundo todo muy limpio",
+      "before": "Antes",
+      "after": "Despues"
```

Comment:
You are missing punctuation here "Después", please review the orthography of all the labels